store/boltdb/boltdb.go

```diff
@@ -10,9 +10,9 @@ import (
 	"sync/atomic"
 	"time"
 
-	"github.com/boltdb/bolt"
 	"github.com/docker/libkv"
 	"github.com/docker/libkv/store"
+	bolt "go.etcd.io/bbolt"
```

Comment:
nit: Was the alias needed? I think it worked without

Reply:
Ah, the package name in the new repo is `bbolt`, not `bolt`. I'm following the import recommendation at https://github.com/etcd-io/bbolt#installing